### PR TITLE
creating db structure

### PR DESCRIPTION
**Pull request by lucyholubova**

### CODE REVIEW

**Review by mporam — APPROVED** [review]

File: db/portfolio.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+# Dump of table portfolio
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `portfolio`;
+
+CREATE TABLE `portfolio` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` int(2) NOT NULL,
+  `url` varchar(255) NOT NULL DEFAULT '',
+  `coment` varchar(255) DEFAULT NULL,
```
Comment (mporam) [inline db/portfolio.sql]: comment has 2 m's

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Comment by alecorra** [comment]
for me it's ok.

File: index.php [hunk 2]
```diff
@@ -1,5 +1,8 @@
-<!DOCTYPE html>
+<?php
+    require('project2/fetch.php');
```
Comment (mporam) [inline index.php]: rename folder project2 - it doesnt mean anything?

File: index.php [hunk 3]
```diff
@@ -53,9 +56,12 @@ <h2 class="whoami"> WHO AM I ?</h2>
 			<h2 class="work">
 				<a href="#contact">WORK</a>
 				<div class="numbers">
-					<a href="#" class="options1"> 1 </a>
-					<a href="#" class="options2"> 2 </a>
-					<a href="#" class="options3"> 3 </a>
+                    <?php
+                    foreach ($result as $project) {
```
Comment (mporam) [inline index.php]: what if `$result` is empty?

**Review by mporam — CHANGES_REQUESTED** [review]

File: portfolio.edit/portfolio.php [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (mporam) [inline portfolio.edit/portfolio.php]: remove dots from directory names please

**Review by mporam — CHANGES_REQUESTED** [review]

**Review by mporam — APPROVED** [review]

**Review by mporam — APPROVED** [review]

File: index.php (hunk 2 above)
Comment (lucyholubova) [reply index.php]: done

File: portfolio-edit/Panel.php [hunk 5]
```diff
@@ -0,0 +1,76 @@
+<?php
+$db = new PDO('mysql:host=192.168.20.20;dbname=Project_2', 'root', '');
+$db-> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$sql = ' SELECT `name`, `url` FROM `portfolio`;';
+$query = $db->query($sql);
+$result = $query->fetchAll();
```
Comment (snack-attack) [inline portfolio-edit/Panel.php:6]: I would name this $projects instead of $result so it's more semantic

File: portfolio-edit/Panel.php [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+        <tr>
+            <th> Projects </th>
+            <th> URL </th>
+            <th> Edit </th>
+            <th> Delete</th>
+        </tr>
+    </thead>
+    <tbody>
+    <?php
+    foreach($result as $project){
```
Comment (snack-attack) [inline portfolio-edit/Panel.php:27]: If you rename $result to $projects - this loop will make more sense
foreach($projects as $project)...

**Pull request merged**